src/modules/units/components/UnitPage.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Paper } from '@mui/material';
+import React, { useEffect, useMemo } from 'react';
+import CommonTabs from '@/components/elements/CommonTabs';
+import Loading from '@/components/elements/Loading';
+import PageTitle from '@/components/layout/PageTitle';
+import NotFound from '@/components/pages/NotFound';
+import useSafeParams from '@/hooks/useSafeParams';
+
+import PrioritizedClientsTable from '@/modules/ce/components/unit/PrioritizedClientsTable';
+import UnitReferralStatus from '@/modules/ce/components/UnitReferralStatus';
+import { useProjectDashboardContext } from '@/modules/projects/components/ProjectDashboard';
+import UnitOverview from '@/modules/units/components/UnitOverview';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import { useGetUnitQuery } from '@/types/gqlTypes';
+
+interface Props {}
+const UnitPage: React.FC<Props> = ({}) => {
+  const { unitId } = useSafeParams() as {
+    unitId: string;
+    projectId: string;
+  };
+
+  const { project, overrideBreadcrumbTitles } = useProjectDashboardContext();
+
+  const {
+    data: { unit } = {},
+    loading,
+    error,
+  } = useGetUnitQuery({
+    variables: {
+      id: unitId,
+      includeCeFields: true, // we only render this page if CE is enabled
+    },
+  });
+
+  // Set the breadcrumb so it says the correct name of this unit and group
+  useEffect(() => {
+    if (!unit) return;
+
+    overrideBreadcrumbTitles({
+      [ProjectDashboardRoutes.UNIT]: unit.name,
+      // fixme unable to inject unit group name
+      [ProjectDashboardRoutes.UNIT_GROUP]: unit.unitGroup?.name || 'Unit Group',
+    });
+  }, [overrideBreadcrumbTitles, unit]);
+
+  const tabDefinitions = useMemo(() => {
+    if (!unit) return [];
+    const defs = [];
+
+    defs.push({
+      title: 'Overview',
+      key: 'overview',
+      contents: <UnitOverview unit={unit} />,
+    });
+
+    const opportunity = unit.latestOpportunity;
+    if (opportunity && project.access.canViewPrioritizedClientLists) {
```

Comment:
Maybe we want to also hide the "Eligible Clients" tab for an opportunity that's already closed? I was clicking around and thought it could be confusing to show, especially under that name, since it's not really true anymore that those clients are "eligible" for this opportunity.

On the other hand, I guess I could imagine someone wanting to find out "who else _was_ eligible for this opportunity at the time?"